app/presenters/two_factor_authentication/webauthn_platform_selection_presenter.rb

```diff
@@ -17,12 +17,12 @@ def render_in(view_context, &block)
       )
     end
 
-    def disabled?
-      user&.webauthn_configurations&.where(platform_authenticator: true)&.any?
+    def single_configuration_only?
+      true
     end
 
     def mfa_configuration_count
-      user.webauthn_configurations.where(platform_authenticator: true).count
+      user&.webauthn_configurations&.where(platform_authenticator: true).count || 0
```

Comment:
Maybe for a follow-up, but it's rather unexpected to me that we need to be `nil`-tolerant on the `users` all of a sudden. It feels to me like maybe it's an issue with how we initialize the specs, but the setup presenters should ideally enforce that a user always exists.